java/lib/com/google/protobuf/protobuf-java/maven-metadata-local.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+  <groupId>com.google.protobuf</groupId>
+  <artifactId>protobuf-java</artifactId>
+  <versioning>
+    <release>2.6.1</release>
+    <versions>
+      <version>2.6.1</version>
+    </versions>
+    <lastUpdated>20151102180249</lastUpdated>
```

Comment:
What time zone is this supposed to be? UTC? I wonder why it's not in epoch seconds.